test/core/channelz/channelz_test.cc

```diff
@@ -283,6 +283,39 @@ TEST_P(ChannelzChannelTest, BasicChannel) {
   ValidateChannel(channelz_channel, {0, 0, 0});
 }
 
+class TestDataSource final : public DataSource {
+ public:
+  using DataSource::DataSource;
+  void AddJson(Json::Object& object) {
```

Comment:
I think this needs to be declared as `override`.

Reply:
done